stages/org.osbuild.shell.init

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+"""
+Write systemwide initialization files, executed for login shells.
+
+Write files to /etc/profile.d with key-value pairs to be set in the shell environment.
+Each file written is sourced by the login shell on initialization.
+"""
+import sys
+import os
+
+import osbuild.api
+
+
+SCHEMA_2 = r"""
+"options": {
+  "properties": {
+    "patternProperties": {
+      "^[\\w.-_]{1,250}$": {
```

Comment:
From what I've read, no specific extension is required, but I wonder whether we would want to enforce e.g. `.sh` for files that should be created? 🤔

Reply:
Every file I've seen in `/etc/profile.d` in my experience has a `.sh` extension but since it's just convention, I wouldn't want to restrict it.

Reply:
To make sure, I ran
```
 dnf whatprovides "/etc/profile.d/*"
```
on F37 and I'm also seeing files with `.csh` and `.local` extensions.

Reply:
I did the same 😄 I'm fine with keeping it as is 😉